src/schema/artist/index.js

```diff
@@ -1,24 +1,48 @@
 // @ts-check
 
-import { pageable, getPagingParameters } from "relay-cursor-paging"
-import { assign, compact, defaults, first, omit, includes } from "lodash"
-import { exclude } from "lib/helpers"
+import {
```

Comment:
My editor is changing all of these spacings (not sure if something else has changed recently). It could be that my local setup is badly borked. The tiny column width it's leaving me is laughable, even on my small laptop.

Additionally, almost this entire file is underlined 'red' because of some TS thing...can't figure out whats going on. It's almost impossible to meaningfully edit this file anymore b/c of these development issues ^ cc @orta not sure why my local setup is always so messed up.

Reply:
IMO, it's worth removing the `// @ts-check` from this page, I added it when I was actively working in MP and could move it closer to working in typescript, but that's not really reality. 

No idea about the line length though, ideally the git commit hook should be making these all 80 chars

Reply:
Thank you for pointing me to `// @ts-check`!

That took care of the red underlines.

Reply:
The thing I don't get with line length is why there are so many changes to this file now? Also, some of the lines that it split up, would be less than 80 if it was all on one line (like some `import` statements at the top). Something feels off in the setup here.